autoload/wiki/page.vim

```diff
@@ -68,6 +68,14 @@ function! wiki#page#rename(newname) abort "{{{1
     return
   endif
 
+  " The target directory must exist
+  let l:target_dir = fnamemodify (expand (l:newpath), ':h')
```

Comment:
Don't add spaces before `(` in the function calls.

Reply:
This will be painful, but will comply :)

Reply:
Haha :)

[ot]
That said; I find it much more important to follow local code styles when working with code than to argue my personal preferences. Of course, when I'm the main author I can decide the styles and obviously choose those that I like myself.
[/ot]